### DISCUSSION TITLE
How to get Google Map Coordinates On An .xml file Uploaded to OsmAnd Application

### DISCUSSION THREAD

**Discussion by JackZ295** [osmandapp/OsmAnd#13853] [Q&A]
Hi! I am trying to get Google Earth coordinates loaded onto a map layer on the OsmAnd application. Is there a way I can go about doing so? Also, is there a particular format that the file with my coordinates should be in? Currently, I have the coordinates saved as an .xml file. I have also opened up the .xml file in Excel and saved the coordinates as an .xlsx file. Here is a screenshot of what the .xlsx file look like: 

![Google_Earth_Coordinates](https://user-images.githubusercontent.com/63502382/156798656-5f16c332-ab42-4487-bb40-cb8fcf781be4.png)

The columns marked off in red are the longitude and latitude coordinates. 

Any input regarding this would be much appreciated. Thanks for reading!

**Comment by Lee-Carre** [comment]
OsmAnd can import GPX files.
